bundles/org.openhab.binding.webexteams/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="webexteams"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Account Thing Type -->
+	<thing-type id="account">
+		<label>WebexTeams Account</label>
+		<description>WebexTeams account used to send messages.</description>
+
+		<channels>
+			<channel id="botname" typeId="botname"/>
+			<channel id="status" typeId="status"/>
+			<channel id="lastactivity" typeId="lastactivity"/>
+		</channels>
+
+		<properties>
+			<property name="personType">Unknown type</property>
+			<property name="name">Unknown name</property>
```

Comment:
After checking the code, this property is identical to the `botname` channel.
At least you have to choose between a property or a channel.